static/js/components.js

```diff
@@ -0,0 +1,96 @@
+Vue.component(VueQrcode.name, VueQrcode)
+
+Vue.component('satspay-paid', {
+  props: ['charge', 'timer'],
+  template: `
+    <div class="col-sm-10 col-lg-8">
+        <q-icon name="check" style="color: green; font-size: 21.4em" class="fit"></q-icon>
+        <div class="row text-center q-mt-lg">
+        <div class="col text-center">
+            <q-btn outline v-if="charge.completelink" :loading="charge.paid" :percentage="timer" type="a"
```

Comment:
- previously it was:
```html
<q-btn outline v-if="charge.webhook" 
```
- is the change on purpose?

Reply:
yes it was wrong